user/templates/windows/qubes-windows-tools-4-1.md

```diff
@@ -277,7 +277,7 @@ Now you should have a menu `Disposable: <VMname>` containing the applications th
 
 For further information on usage of disposables, see [How to use disposables](/doc/how-to-use-disposables/).
 
-**Caution:** *If a Windows-based disposable is used from another qube via the Open/Edit in DisposableVM command, this disposable may not close automatically, due to the command prompt window still running in this dispvm. In this case, the disposable has to be shut down  manually.*
+**Caution:** *If a Windows-based disposable is used from another qube via the* `Open/Edit in DisposableVM` *command, this disposable may not close automatically, due to the command prompt window still running in this dispvm. In this case, the disposable has to be shut down  manually.*
```

Comment:
Here I just splitted the sentence in parts that need an emphasis or highlight so that i can go around the rst problem i mentioned earlier.
for example now it looks like : https://current-qubes-docrtd.readthedocs.io/en/rtd-deploy-pr/user/templates/windows/qubes-windows-tools-4-1.html
@marmarek FYI